handbook/engineering/search/goals.md

```diff
@@ -2,64 +2,52 @@
 
 ## Goals
 
-Goals help us achieve our [vision](index.md#vision) for search.
+### Support large scale organizations
 
-### Mature query language
+Timeline: ~6 months (May 2021)
 
-**Problem:** The Sourcegraph query language does not support the growing complexity of searches being run by our users.
+#### Definition of large scale
 
-Examples:
+Organizations with:
 
-- I want to only show examples from repositories that have been modified in the last 3 months so I see the most recent solutions.
-- I want to find repositories that have not been modified in over a year so I can deprecate them.
-- TODO: add more examples.
+- A Git directory size between 100 GB and 50 TB
+- Total lines of code (LOC) between 1 billion and 5 billion
 
-**Success criteria**
+See [looker](https://sourcegraph.looker.com/looks/436) for current customer data.
 
-TODO table of use cases / capabilities we want to achieve, comparing Sourcegraph with other search tools
+#### Problem
 
-**Planned work**
+Large customers expect search to be fast at their scale. Customers expect to be able to search over their codebase intuitively, and want their first search result quickly. For larger organizations, customers care about getting all search results for important changes in their codebase. Supporting search at this scale also helps work towards supporting the Cloud growth efforts.
```

Comment:
Rather than talking about expectations, let's talk about the current experience, listing current metrics. 

For example, something like the following (but worded better to sound more fluid and like the story of the current experience).

```
Customers with more than X LOC experience timeouts when performing searches without limiting total repositories. Currently, structural search only works when scoped to < 50 repositories. 
```